modules/ocf_backups/files/upload-to-box

```diff
@@ -217,6 +218,9 @@ def main(args):
     with open('/opt/share/backups/box-creds.json') as f:
         creds = json.load(f)
 
+    # Make sure our refresh token is still active, and fail otherwise.
+    get_access_token(creds)
```

Comment:
I'd say putting a `try`/`except` or similar around the access token and sharing part below would be enough here, since then the backups would still be getting made but a warning could be put into the email if the token is bad (as mentioned on IRC by dkessler).